OrbitGl/TextRenderer.cpp

```diff
@@ -166,8 +166,16 @@ void TextRenderer::AddTextInternal(texture_font_t* font, const char* text, const
   float strWidth = 0.f;
   int minX = INT_MAX;
   int maxX = -INT_MAX;
+  constexpr GLuint indices[6] = {0, 1, 2, 0, 2, 3};
```

Comment:
Nit: I am not sure, if we have any convention yet. Our clang-tidy settings, complains about c-style arrays and suggest to use std::array instead. I don't have strong opinion on this, but just wanted to mention this.

Reply:
sure, changed it to a std::array

Reply:
Okay, thanks!